src/ui/public/agg_types/buckets/date_histogram.js

```diff
@@ -41,7 +41,8 @@ export default function DateHistogramAggType(timefilter, config, Private) {
     makeLabel: function (agg) {
       const output = this.params.write(agg);
       const params = output.params;
-      return params.field + ' per ' + (output.metricScaleText || output.bucketInterval.description);
+      const field = params.field || agg.params.field.displayName || '';
```

Comment:
I worry about accessing the field display name directly like this because elsewhere in this file we access params information in a particularly defensive way: https://github.com/elastic/kibana/pull/8638/files#diff-2cd930ccd69e832107c9578ef648e5b7R23

That leads me to believe that there are circumstances where `agg` doesn't have the nested params data, which would cause this code to trigger an error.